k8s-install/scripts/install-cni.sh

```diff
@@ -125,6 +125,7 @@ sed -i s~__ETCD_CERT_FILE__~${CNI_CONF_ETCD_CERT:-}~g $TMP_CONF
 sed -i s~__ETCD_KEY_FILE__~${CNI_CONF_ETCD_KEY:-}~g $TMP_CONF
 sed -i s~__ETCD_CA_CERT_FILE__~${CNI_CONF_ETCD_CA:-}~g $TMP_CONF
 sed -i s~__ETCD_ENDPOINTS__~${ETCD_ENDPOINTS:-}~g $TMP_CONF
+sed -i s~__LOG_LEVEL__~${LOG_LEVEL}~g $TMP_CONF
```

Comment:
Yeah, I understand why you would say that, I intentionnally put it in the Dockerfile because of how I use docker images in general: ervery time I use a docker image that wasn't made by made, I check the Docker file to see which port it exposes and which env variables are available to me to configure at run time. That's the reason I do this but I can change this if you want to :)
Thanks